assets/src/application/ui/input/EspressoButton/EspressoButton.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import classNames from 'classnames';
 import { Button, Tooltip } from 'antd';
-import Icon from '@ant-design/icons';
+import AntDIcon from '@ant-design/icons';
```

Comment:
`AntDIcon` looks a bit weird :smiley:

Reply:
`Icon` was needed as well :)

Reply:
my initial thought was `AntIcon`
:ant:`Icon`

Reply:
the :ant: is on the :fire:

Reply:
Done :)